repos/fdbt-site/src/pages/returnService.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { ReactElement } from 'react';
+import upperFirst from 'lodash/upperFirst';
+import { ErrorInfo, NextPageContextWithSession, ServiceType } from '../interfaces';
+import FormElementWrapper from '../components/FormElementWrapper';
+import TwoThirdsLayout from '../layout/Layout';
+import {
+    OPERATOR_ATTRIBUTE,
+    MATCHING_JSON_ATTRIBUTE,
+    MATCHING_JSON_META_DATA_ATTRIBUTE,
+} from '../constants/attributes';
+import { getPassengerTypeById, getServicesByNocCodeAndDataSource } from '../data/auroradb';
+import ErrorSummary from '../components/ErrorSummary';
+import { getAndValidateNoc, getCsrfToken, isReturnTicket } from '../utils';
+import CsrfForm from '../components/CsrfForm';
+import { getSessionAttribute } from '../utils/sessions';
+import { redirectTo } from '../utils/apiUtils';
+import InformationSummary from '../components/InformationSummary';
+import BackButton from '../components/BackButton';
+
+const title = 'Return Service - Create Fares Data Service';
+const description = 'Service selection page of the Create Fares Data Service';
```

Comment:
Think this needs updating to 'Return service selection page....'

Reply:
added